twilight-gateway/src/shard.rs

```diff
@@ -281,7 +281,7 @@ enum NextAction {
 ///
 /// Because shards should not be used across multiple tasks it's not always easy
 /// to directly send [gateway commands] over a shard. As a convenience method,
-/// [`Shard::sender`] can be used to receive an MPSC channel sender which, in
+/// [`Shard::queue`] can be used to receive an MPSC channel sender which, in
```

Comment:
This obviously needs rewording, but if you don't mind I'd like to do that myself.

Reply:
I'll leave the unresolved reference here as you'll be updating the docs anyway.